### PR TITLE
Update to use value prop of examples object

### PR DESCRIPTION
**Pull request by RockslideJr**
This PR fixes the bug in [API-5709](https://vajira.max.gov/browse/API-5709). It also includes some small refactoring to reduce complexity

### CODE REVIEW

File: src/utilities/oas-schema/oas-schema.ts [hunk 1]
```diff
@@ -44,17 +44,17 @@ class OASSchema {
...
-    // e.g. [ 'findFormByFormName', { form_name: 'VA10192' } ]
-    const operationIdToParameters = methods.map((method) => {
-      // transforms each OAS parameter into a tuple that contains the parameter name and example value
-      // e.g. [form_name: 'VA10192']
+    return methods.reduce((parameterExamples, method) => {
+      // transforms each OAS parameter into an object that contains the parameter name and example value
       const requiredParametersAndExamples = method.parameters
         .filter((parameter) => parameter.required)
-        .map((parameter) => {
+        .reduce((requiredExampls, parameter) => {
```
Comment (alexandervalencia) [inline src/utilities/oas-schema/oas-schema.ts:51]: typo:
```suggestion
        .reduce((requiredExamples, parameter) => {
```

File: src/utilities/oas-schema/oas-schema.ts [hunk 2]
```diff
@@ -44,17 +44,17 @@ class OASSchema {
...
           const { name, example } = parameter;
-          return [name, example];
-        });
+          return {
+            ...requiredExampls,
```
Comment (alexandervalencia) [inline src/utilities/oas-schema/oas-schema.ts:54]: typo:
```suggestion
            ...requiredExamples,
```

File: src/utilities/oas-schema/oas-schema.ts [hunk 3]
```diff
@@ -72,30 +72,30 @@ class OASSchema {
...
 
-      return [method.operationId, parameters];
-    });
-
-    return Object.fromEntries(operationIdToParameters);
+      return {
+        ...parameterExamples,
+        [method.operationId]: parameters,
+      };
+    }, {});
```
Comment (alexandervalencia) [inline src/utilities/oas-schema/oas-schema.ts:98]: `getParameters` has become pretty long, at nearly 50 lines now. Not necessarily part of this PR, but what are your thoughts on a ticket to reduce some of the complexity here by breaking up this function?

**Review by alexandervalencia — COMMENTED** [review]
a few typos and a question for you

File: src/utilities/oas-schema/oas-schema.ts (hunk 3 above)
Comment (RockslideJr) [reply src/utilities/oas-schema/oas-schema.ts:98]: that's a great callout! Christine and I have actually been talking through a general refactoring of some stuff that will hopefully help reduce complexity

**Review by alexandervalencia — APPROVED** [review]
LGTM! 🚀 have a good weekend!

**Pull request merged**